File: 6-kyu/multiples-of-3-or-5/c++/solution.cpp
int solution(int number) 
{
    if (number <= 3) return 0;
    else if (number < 5) return 3;
    int ret = 0;
    for (unsigned i = 0; i < number; i++) {
        if (i % 3 == 0 || i % 5 == 0) ret += i;
    }
    return ret;
}

File: 6-kyu/multiples-of-3-or-5/typescript/solution.ts
from functools import reduce

def solution(number):
    if number <= 3:
        return 3 if number == 3 else 0
    return reduce(lambda acc, num: acc + num, list(filter(lambda item: item % 3 == 0 or item % 5 == 0, list(range(3, number)))))